tests/functional/test_examples.py

```diff
@@ -16,28 +16,29 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with palace.  If not, see <https://www.gnu.org/licenses/>.
 
+from os.path import abspath, basename, join
 from subprocess import run
 from sys import executable
 
-EVENT = './palace-event.py'
-HRTF = './palace-hrtf.py'
-INFO = './palace-info.py'
-LATENCY = './palace-latency.py'
-REVERB = '/palace-reverb.py'
-STDEC = './palace-stdec.py'
-TONEGEN = '/palace-tonegen.py'
+EVENT = join(abspath(basename('./palace-event.py')))
+HRTF = join(abspath(basename('./palace-hrtf.py')))
+INFO = join(abspath(basename('./palace-info.py')))
+LATENCY = join(abspath(basename('./palace-latency.py')))
+REVERB = join(abspath(basename('./palace-reverb.py')))
+STDEC = join(abspath(basename('./palace-stdec.py')))
+TONEGEN = join(abspath(basename('./palace-tonegen.py')))
```

Comment:
HAI

Should have said it clearer, I meant `join(abspath(basename(__file__)), ...)`.

Please do the same for the audio.

KTHXBYE

Reply:
Wouldn't `basename(__file__)` return `test_examples.py`? I'm confused.

Reply:
OK I will use `dirname(__file__)` instead since that's the only way that would make sense to me.

Reply:
LOL yes, my mistake!